content/docs/hosting/BYOC.md

```diff
@@ -98,169 +99,178 @@ With Appbase.io Dashboard you can easily share cluster with your team members. E
 
-Here are the steps that you can follow to deploy Get Arc ID:
+Follow the steps listed below to successfully create an Arc instance.
 
--   **Step 1 -** Go to [Arc Dashboard](https://arc-dashboard.appbase.io/install).
+**Step 1 -** Go to [Arc Dashboard](https://arc-dashboard.appbase.io/install).
 
 ![](https://i.imgur.com/YZubabh.png)
 
--   **Step 2 -** Enter your email address
+**Step 2 -** Enter your email address
 
--   **Step 3 -** You will receive an OTP on entered email address. Enter OTP to verify the email address
+**Step 3 -** You will receive an OTP on entered email address. Enter OTP to verify the email address
 
--   **Step 4-** You will receive an email with `ARC_ID` which we can use while creating EC2 machine.
+**Step 4-** You will receive an email with `ARC_ID` which can be used with Arc configuration.
 
-Here are the steps that you can follow to install Arc Middleware using AMI
+---
+
+### Using AMI
 
-**Step 1 -** Select ARC [Amazon Machine Image](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) from AWS Marketplace
+Now you can deploy Arc using Amazon Machine Image (AMI) on the AWS Marketplace. With the AMI, you can Arc with one click on an AWS EC2 instance.
+
+Here are the steps that you can follow to install Arc using AMI
+
+**Step 1 -** Create an [Arc instance](/docs/hosting/BYOC/#how-to-create-arc-instance).
+
+**Step 2 -** Select Arc AMI from [AWS Marketplace]()
 
 // TODO Add Marketplace Link + image of marketplace
 
-**Step 2 -** Select EC2 Instance Type required for Deploying Arc. We recommend using `t2 medium`.
+**Step 3 -** Select EC2 Instance based on your traffic for Deploying Arc. We recommend using [t2 medium](https://aws.amazon.com/ec2/pricing/on-demand/).
 
-**Step 3 -** Enter env variables in `User Data` Section while configuring `EC2` instance. Here is sample script which you can enter in `User Data` Section.
+That's all we need to follow in order to deploy an Arc cluster 🚀. Once the EC2 machine is created and _running_ successfully, you can get Public DNS / IP address from the EC2 dashboard. This URL becomes your end point for accessing the Arc services. You can also point your domain to this DNS or IP.
 
-```bash
-#!/bin/bash
-#!/bin/bash
-echo ES_CLUSTER_URL='https://search-test.us-east-1.es.amazonaws.com' >> /etc/systemd/system/arc.env
+## ![](https://i.imgur.com/PDs8DK0.png)
 
-echo USERNAME='admin' >> /etc/systemd/system/arc.env
-echo PASSWORD='admin' >> /etc/systemd/system/arc.env
-echo ARC_ID='YOUR ARC ID' >> /etc/systemd/system/arc.env
-```
+> **Note:** We highly recommend using cluster with [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) certificate, as you will be using this cluster URL on [Arc Dashboard](https://arc-dashboard.appbase.io) which is secured with TLS certificate. So to avoid unsafe script errors, it is better to setup TLS for your domain or EC2 public DNS.
 
-![](https://i.imgur.com/T00tC8W.png)
+#### **Adding TLS Certificate**
```

Comment:
@lakhansamani This section should be arranged below the `Configuring Arc` section.